src/components/RouteActions.tsx

```diff
@@ -35,7 +35,7 @@ interface RouteActionsProps {
 }
 
 const RouteActions: VoidComponent<RouteActionsProps> = (props) => {
-  const [routeResource] = createResource(() => props.routeName, getRoute)
+  const [routeResource] = createResource(() => props.routeName, getRoute)  // wtf with the duplicate queries, why not pass route in?
```

Comment:
@incognitojam do you know why we sometimes pass route and other times pass routeName for the child component (?) to re-query the route when the parent had it? Seems confusing